subprojects/instant-execution/src/main/kotlin/org/gradle/instantexecution/DefaultInstantExecution.kt

```diff
@@ -78,6 +79,10 @@ class DefaultInstantExecution(
         host.newStateSerializer()
     }
 
+    private
+    val buildOperationExecutor =
+        host.getService(BuildOperationExecutor::class.java)
```

Comment:
❌ Rebase so you can write it as `host.service<BuildOperationExecutor>()`.

Reply:
Done